ipatests/test_cmdline/test_ipagetkeytab.py

```diff
@@ -201,6 +201,47 @@ def test_5_use_disabled(self, test_service):
         except Exception as errmsg:
             assert('Unable to bind to LDAP. Error initializing principal' in str(errmsg))
 
+    def test_6_quiet_mode(self, test_service):
+        """
+        Try to use quiet mode
+        """
+        test_service.ensure_exists()
+        # getkeytab without quiet mode option enabled
+        result = self.run_ipagetkeytab(test_service.name)
+        err = result.error_output.split("\n")[0]
+        assert err == f"Keytab successfully retrieved and stored in:" \
+                      f" {self.keytabname}"
+        assert result.returncode == 0
+
+        # getkeytab with quiet mode option enabled
+        result1 = self.run_ipagetkeytab(test_service.name, args=tuple("-q"))
+        assert result1.returncode == 0
+
+    def test_7_server_name_check(self, test_service):
+        """
+        Try to use -s for server name
+        """
+        test_service.ensure_exists()
+        self.assert_success(test_service.name, args=["-s", api.env.host])
+
+    def test_8_keytab_name_check(self, test_service):
+        """
+        Try to use -k keytab name
+        """
+        test_service.ensure_exists()
+        self.assert_success(test_service.name, args=["-k", self.keytabname])
+
+    def test_9_keytab_encryption_check(self, test_service):
+        """
+        Try to use -e for different types of encryption check
+        """
+        encryptes_list = [
+            "aes256-cts-hmac-sha1-96",
+            "aes128-cts-hmac-sha256-128",
+        ]
+        for encryptes in encryptes_list:
+            self.assert_success(test_service.name, args=["-e", encryptes])
```

Comment:
Instead of re-running `ipa-getkeytab` with single enctype each time, please change this to
```
    self.asser_success(test_service.name, args=["-e", ",".join(encryptes_list))
```
without a loop. `-e` accepts list of encryption types separated by a comma.